scaffold/github/workflows/PantheonReviewAppsManual.yml

```diff
@@ -0,0 +1,72 @@
+name: "Pantheon Review Apps"
+
+on:
+  pull_request:
+    types: [ labeled, unlabeled, opened, synchronize, reopened]
```

Comment:
```suggestion
    types: [ labeled, opened, synchronize, reopened]
```

Reply:
committed in https://github.com/Lullabot/drainpipe/pull/670/commits/ab80299864a0713b181a1e19a3a0b9e7d35b018f